dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -460,57 +544,67 @@ public void run(){
                                     }
                                 }
 
-                                if(conflictPresent) {
+                                if (conflictPresent) {
                                     //Setting the process status
-                                    setStatus( session, endpointId, ProcessStatus.FINISHED, null );
+                                    setStatus(session, endpointId, ProcessStatus.FINISHED, null);
                                 } else {
                                     String noConflictMessage;
                                     try {
-                                        noConflictMessage = LanguageUtil.get( loggedUser.getLocale(), "push_publish_integrity_conflicts_not_found" );
+                                        noConflictMessage = LanguageUtil.get(
+                                                loggedUser.getLocale(),
+                                                "push_publish_integrity_conflicts_not_found");
                                     } catch ( LanguageException e ) {
                                         noConflictMessage = "No Integrity Conflicts found";
                                     }
                                     //Setting the process status
-                                    setStatus( session, endpointId, ProcessStatus.NO_CONFLICTS, noConflictMessage );
+                                    setStatus(session, endpointId, ProcessStatus.NO_CONFLICTS, noConflictMessage);
                                 }
-
                             } else if ( response.getStatus() == HttpStatus.SC_PROCESSING ) {
-
-                                continue;
-                            } else if ( response.getStatus() == HttpStatus.SC_RESET_CONTENT ) {
+                                // do nothing
+                            } else if (response.getStatus() == HttpStatus.SC_RESET_CONTENT) {
                                 processing = false;
                                 //Setting the process status
-                                setStatus( session, endpointId, ProcessStatus.CANCELED, null );
+                                setStatus(session, endpointId, ProcessStatus.CANCELLED, null);
                             } else {
-                                setStatus( session, endpointId, ProcessStatus.ERROR, null );
-                                Logger.error( this.getClass(), "Response indicating a " + response.getStatusInfo().getReasonPhrase() + " (" + response.getStatus() + ") Error trying to retrieve the Integrity data from the Endpoint [" + endpointId + "]." );
+                                setStatus(session, endpointId, ProcessStatus.ERROR, null);
+                                Logger.error(
+                                        this.getClass(),
+                                        "Response indicating a " + response.getStatusInfo().getReasonPhrase()
```

Comment:
Codacy found an issue: [The String literal "Response indicating a " appears 4 times in this file; the first occurrence is on line 501](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)